src/main/readout.go

```diff
@@ -33,7 +33,7 @@ import (
 	"math"
 	"os"
 	"path/filepath"
-	"github.com/nDenerserve/SmartPi/src/smartpi"
```

Comment:
Annoying, yes and most of all unnecessary.
What is the benefit of this approach?
The old way I could just clone (my fork) and build via make.
Now, I have to fix all these unnecessary stuff without any benefit.